include/knowhere/expected.h

```diff
@@ -80,6 +81,8 @@ Status2String(knowhere::Status status) {
             return "raft inner error";
         case knowhere::Status::invalid_binary_set:
             return "invalid binary set";
+        case knowhere::Status::invalid_instruction_set:
+            return "current index not supported in this cpu";
```

Comment:
"the current CPU model lacks hardware instructions that are needed for the implementation of the chosen index type"